src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/ContextIntervalTree.cs

```diff
@@ -119,9 +126,9 @@ private T GetSmallestContainingIntervalWorker(int start, int length, Func<T, int
                     if (predicate(currentNode.Value, start, length))
                     {
                         // hold onto best answer
-                        if (EqualityComparer<T>.Default.Equals(result, default) ||
-                            (Introspector.GetStart(result) <= Introspector.GetStart(currentNode.Value) &&
-                             Introspector.GetLength(currentNode.Value) < Introspector.GetLength(result)))
+                        if (EqualityComparer<T/*??*/>.Default.Equals(result!, default!) ||
```

Comment:
lol.

Reply:
(both because i get what you're doing, and because it looks like you wrote `??` to indicate confusion :))